source/_integrations/agent_dvr.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: Agent DVR
+description: Access and control Agent DVR from Home Assistant.
+logo: agent_dvr.png
```

Comment:
Please open up a PR to our brands repository to add the logo & icon of the brand, and remove the `logo` directive from this PR.

<https://github.com/home-assistant/brands>

Be sure to add link between the 2 PR's in the openings post of both.